modules/javafx.swt/src/test/java/test/javafx/embed/swt/SWTCursorsTest.java

```diff
@@ -33,19 +33,20 @@
 import org.eclipse.swt.SWT;
 import org.eclipse.swt.widgets.Display;
 import org.eclipse.swt.widgets.Shell;
-import org.junit.Rule;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
 
-import static org.junit.Assert.assertNotNull;
+import java.util.concurrent.TimeUnit;
 
-public class SWTCursorsTest {
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 
-    @Rule
-    public SwtRule ctx = new SwtRule();
```

Comment:
this looks like a breaking change - what would be the junit5 equivalent of `@Rule`?

Reply:
`@ExtendWith`, but I think there is no equivalent of a method rule. In any case, another SWT test seems to be just fine without it, so I adapted the pattern to all other tests. We need to check if that works though.

Reply:
So this was interesting, turns out it wasn't that stable after all even before!
With some adjustments to the Gradle file, I could make the tests work and they are now green.